api_tests/src/image.spec.ts

```diff
@@ -28,13 +27,16 @@ import {
   setupLogins,
   waitForPost,
   unfollows,
+  editPostThumbnail,
+  getPost,
+  waitUntil,
 } from "./shared";
 const downloadFileSync = require("download-file-sync");
 
 beforeAll(setupLogins);
 
-afterAll(() => {
-  unfollows();
+afterAll(async () => {
+  await unfollows();
 });
```

Comment:
You may be able to get away with:
```typescript
afterAll(unfollows);
```
in place of what you're currently doing. It looks like the problem before was that you were calling an async function from a synchronous function. My suggestion above will make it just call the async function without the need for an anonymous async function wrapping it. This applies to the other spec files where you use this pattern as well.